tests/baselines/reference/jsdocTypeTagCast.types

```diff
@@ -209,4 +209,18 @@ if(/** @type {numOrStr is string} */(numOrStr === undefined)) { // Error
 }
 
 
+var asConst1 = /** @type {const} */(1);
+>asConst1 : 1
+>(1) : 1
+>1 : 1
 
+var asConst2 = /** @type {const} */({
+>asConst2 : { x: number; }
+>({    x: 1}) : { x: number; }
+>{    x: 1} : { x: number; }

```

Comment:
This should be `{ readonly x: 1 }` if it's the equivalent of `var asConst2 = { x: 1 } as const`, right?